tests/FakeItEasy.Tests/FakeItEasy.Tests.csproj

```diff
@@ -273,4 +272,4 @@
     </PropertyGroup>
     <Error Condition="!Exists('..\..\packages\StyleCop.MSBuild.4.7.49.1\build\StyleCop.MSBuild.Targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\StyleCop.MSBuild.4.7.49.1\build\StyleCop.MSBuild.Targets'))" />
   </Target>
-</Project>
+</Project>
```

Comment:
I didn't see this in the diff at home, but it looks like the final end of line was removed. I guess `.editorconfig` wasn't respected by the IDE…
I'll have to check my plugin once I get home.